gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
I don't see any addition that replaces the functionality of this removal. We need to arrow to reverse somehow in RTL mode.